htdocs/product/price.php

```diff
@@ -1743,6 +1758,17 @@ function on_change() {
 		print '</td>';
 		print '</tr>';
 
+		// Packaging / Conditionnement
+		if (getDolGlobalString('PRODUCT_USE_CUSTOMER_PACKAGING')) {
+			print '<tr><td>';
+			print $form->textwithpicto($langs->trans("PackagingForThisProduct"), $langs->trans("PackagingForThisProductSellDesc"));
+			print '</td><td>';
+			$packaging = $object->packaging;
+			print '<input class="flat" name="packaging" size="5" value="'.$packaging.'">';
```

Comment:
As $packaging is a float, it may be preferable to apply some rounding before displaying it.